### PR TITLE
Implement a traceless H(curl div) element

### PR DESCRIPTION
**Pull request by pbrubeck**
Reference https://doi.org/10.1093/imanum/drz022

### CODE REVIEW

File: FIAT/gopalakrishnan_lederer_schoberl.py [hunk 1]
```diff
@@ -0,0 +1,154 @@
...
+
+
+class GLSDual(dual_set.DualSet):
+    def __init__(self, ref_el, degree):
+        sd = ref_el.get_spatial_dimension()
+        top = ref_el.get_topology()
+        nodes = []
+        entity_ids = {dim: {entity: [] for entity in sorted(top[dim])} for dim in sorted(top)}
+
+        # Face dofs: bidirectional nt Legendre moments
```
Comment (rckirby) [inline FIAT/gopalakrishnan_lederer_schoberl.py:110]: Is this general dimension (1 normal + (sd-1) tangents)?  If so, the `nt` comment is misleading.  If not and this is just 2, the code is over engineered?

**Review by rckirby — CHANGES_REQUESTED** [review]
Mostly looks fine.  Just an update on the comment (I think)?

File: FIAT/gopalakrishnan_lederer_schoberl.py (hunk 1 above)
Comment (pbrubeck) [reply FIAT/gopalakrishnan_lederer_schoberl.py]: It's for general dimension, I changed the comment to something more accurate: "moments of nt components against a basis for P_{k-1}"

Comment (rckirby) [reply FIAT/gopalakrishnan_lederer_schoberl.py]: But if you still say `nt components` that's a bit ambiguous.  I'd recommend something more like `moments of the normal component and tangential component(s) against a basis for P_{k-1} on the facet`

**Pull request merged**